package.json

```diff
@@ -28,7 +28,7 @@
   "jest": {
     "setupTestFrameworkScriptFile": "<rootDir>/js/tests/setupTests.js",
     "transformIgnorePatterns": [
-      "[/\\\\]node_modules[/\\\\](?!yoast-components|yoastseo|lodash-es).+\\.js$"
+      "[/\\\\]node_modules[/\\\\](?!yoast-components|yoastseo|lodash-es|@yoast).+\\.js$"
```

Comment:
Could you explain why this is necessary, please? I'm not yet aware of what this does for us

Reply:
This makes sure that babel runs on the `@yoast` packages when running unit tests in the plugin. Probably doesn't break if we don't add it, more for the future.